src/app/static/src/app/store/translations/locales.ts

```diff
@@ -23,6 +23,11 @@ export interface Translations {
     cancelEdit: string,
     cancelEditLoggedIn: string,
     cancelFitting: string,
+    capacityAccessLevel: string,
```

Comment:
Are these keys remnants of when you were displaying user role rather than dataset permission? "Capacity" doesn't seem that helpful as a term here. Would `adrAccessLevel`, `adrRead`, `adrReadWrite`, `adrReadTooltip` (or message) and `adrReadWrite` be closer to actual usage?

Reply:
I agree with this - and I don't think we need the admin/member distinction at the dataset level. Would also suggest a common prefix for inline and tooltip text e.g. `adrRead` + `adrReadTooltip`.

Main thing is that we clearly communicate the most important point - that without write access you won't be able to upload your model outputs. This should probably be on the screen - with the tooltip providing some help on how to resolve the issue (e.g. contacting relevant people). We have plenty of space to do this.

Reply:
I tried to put more text on the screen but, it seems more to many text.

Reply:
Sorry, not sure what you mean, do you mean it looks too busy? There's not much else on the baseline screen so I'd be surprised if we couldn't find a way to display that information in a way that doesn't look cluttered. Could use `small` text styling if appropriate.